week1/todoList/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TodoList</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+</head>
+<body>
+    <header class="logo">
+        <h2>Todo List</h2>
+    </header>
+    <nav>
+        <button>오늘만 보기</button>
+        <button>내일만 보기</button>
+        <button>함께 보기</button>
+    </nav>
+    <main>
+        <section class="left">
+            <section class="todo-header">
+                <h3 class="todo-title">오늘 할 일</h3>
+            </section>
+            <footer class="todo-add-group">
+                <input class="todo-add-group__input">
+                <button class="todo-add-group__btn-add"><span class="material-icons">&#xe145;</span></button>
+            </footer>
+        </section>
+        <section class="right">
+            <section class="todo-header">
+                <h3 class="todo-title">내일 할 일</h3>
```

Comment:
여기도 만약에 header라고 쓰고 싶었으면 h3를 감싸는 태그가 <header>태그이어야 적합한 클래스명 같아!
```suggestion
<header class="todo-header">
                <h3 class="todo-title">내일 할 일</h3>
 </header>
```
만약 여기서 class명이 header, title이 의미가 header도 제목, 웹싸이트를 나타내는 로고, 검색 폼, 작성자 이름 등의 요소등이라서 중복된다고 생각한다면,,!

header의 class명만 `todo-title로` 해주고 h3에는 클래스명을 안적어줘도 될것같아!